### ISSUE TITLE
Missing properties cccev:expectedDataType, cccev:expectedValue, cccev:maximumValue, cccev:minimumValue etc.

### ISSUE THREAD

**Issue by jimjyang** [SEMICeu/CCCEV#45]
[The class CriterionRequirement in CCCEV v.1](https://github.com/SEMICeu/CCCEV/blob/master/releases/1.00/core_evidence_and_core_criterion_vocabulary_version_1.0.0.png) had properties expectedDataType, expectedValue, maximumValue, minimumValue etc., which are necessary to specify e.g. the requirement "age (positiveInteger) >= 20" machin-readably.

The class CriterionRequirement in CCCEV v.1 is split into the classes Requirement, Information Requirement and Constraint in [CCCEV v.2](https://github.com/catalogue-of-services-isa/CPSV-AP/blob/master/releases/3.0.0/html/overview.jpg), without explicitly mentioning where the abovementioned (and several other) properties of CriterionRequirement are in v.2. 

Our questions are:

1. Where are the properties of CriterionRequirement in CCCEV v.1 now in CCCEV v.2? If they are intentionally omitted, why? As exemplified above, they are necessary in order to have machin-readable specifications of requirements and they should be standardized at the EU level.
2. The class CriterionRequirement is deprecated, are the properties cccev:expectedDataType (with cccev namespace) etc. also deprecated? Or, is it still allowed and advisable to use them (with new domains), e.g. in connection to (national application profiles of) CPSV-AP?

**Comment by bertvannuffelen** [comment]
@jimjyang indeed there has been a major shift between CCCEV 1.0 and CCCEV 2.0.

The listed original CCCEV 1.0 properties have not been reused as their semantics are semantically conflicting with CCCEV 2.0. In order to have CCCEV 1.0 implementations not to face with a forced semantical switch of the properties they are kept with the original semantics  in the deprecated CCCEV 1.0.

Nevertheless there is in CCCEV 2.0 an alternative approach that facilitates the same use case.
Namely the whole area of the term[Information Concept](https://semiceu.github.io/CCCEV/releases/2.00/#Information%20Concept).
An Information Concept is a piece of information that plays a role in the exchange process between the Requirement and the Evidence. 
The notion of expected value part of this class. And that value allows more than min/max expressions.

I hope this clarifies how the usecases of version 1.0 are included in version 2.0.

**Comment by jimjyang** [comment]
@bertvannuffelen 

Yes, we have already tried to model requirements using the classes Information Concept, Constraint, ... (conf. [the explanation](https://informasjonsforvaltning.github.io/cpsv-ap-no/#%C3%85-beskrive-dokumentasjonskrav) - though in Norwegian only - on how to model a requirement, in our draft version of CPSV-AP-NO)

**Comment by bertvannuffelen** [comment]
@jimjyang  thanks for the pointer to your implementation guide.
